### PR TITLE
[fortinet_fortigate] Add ECS fields

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Add `user.name`, `host.name` and `dns.question.registered_domain` fields.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/fortinet_fortigate/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -155,6 +155,14 @@ processors:
       field: email.subject
       copy_from: fortinet.firewall.subject
       if: "ctx?.fortinet?.firewall?.subject != null"
+  - set:
+      field: host.name
+      copy_from: fortinet.firewall.srcname
+      if: "ctx?.fortinet?.firewall?.srcname != null"
```
Comment (efd6) [inline packages/fortinet_fortigate/data_stream/log/elasticsearch/ingest_pipeline/default.yml:161]:
```suggestion
      if: "ctx.fortinet?.firewall?.srcname != null"
```
(similar throughout)

File: packages/fortinet_fortigate/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -434,6 +442,11 @@ processors:
       value: "{{source.address}}"
       if: "ctx.source?.address != null"
       allow_duplicates: false
+  - append:
+      field: related.hosts
+      value: "{{host.name}}"
```
Comment (efd6) [inline packages/fortinet_fortigate/data_stream/log/elasticsearch/ingest_pipeline/default.yml:447]:
```suggestion
      value: "{{{host.name}}}"
```

**Comment by LaZyDK** [comment]
My mistake.

File: packages/fortinet_fortigate/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.7.1"
```
Comment (efd6) [inline packages/fortinet_fortigate/changelog.yml:2]:
```suggestion
- version: "1.8.0"
```
(enhancements bump minor)

**Review by efd6 — COMMENTED** [review]
Needs an `elastic-package build`.

**Comment by LaZyDK** [comment]
Ready for review

**Comment by LaZyDK** [comment]
should be good now

**Review by efd6 — APPROVED** [review]
Thanks

**Pull request merged**